modules/custom/cu_livechat_bundle/cu_livechat_bundle.info

```diff
@@ -6,6 +6,7 @@ package = CU-LiveChat
 project = cu_livechat_bundle
 project status url = https://github.com/CuBoulder/cu_livechat_bundle
 project_demo_url = http://www.colorado.edu/webcentral/node/1532
+https://websupport.colorado.edu/article/64-live-chat
```

Comment:
This should be a replacement

Reply:
Fixed in commit: https://github.com/CuBoulder/express_mono/pull/243/commits/26983e9d53baa214ba14e0d039ce7b17ef14e89d